src/pages/nominate.js

```diff
@@ -54,12 +52,11 @@ class App extends React.Component {
 
   handleSubmit = e => {
     const validation = this.validateName();
-    if (!validation && window.confirm('This form will take you to Twitter https://twitter.com/ in order to submit your nomination')) {
+    if (!validation && window.confirm('This form will take you to Twitter https://twitter.com/ in order to send your nomination')) {
       const urlEncodedTwitterHandle = encode(this.state.name);
```

Comment:
Looks like we can delete this variable `urlEncodedTwitterHandle ` and `urlEncodedReason` since we're replacing them with the properties in `this.state`

Reply:
True, good point.  The changes here are only from reverting the one commit, so I didn't dig into further changes since I wasn't sure whether we'd retry the encoding changes or just roll back completely.  Given the short timeline currently, I'll look into clearing them out

Reply:
Ahhh gotcha, that makes total sense!